packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.ts

```diff
@@ -22,7 +22,7 @@ export default class BitcoinWalletProvider extends Provider {
     super('BitcoinWalletProvider');
 
     this._network = network;
-    this._unusedAddressesBlacklist = [];
+    this._unusedAddressesBlacklist = {};
```

Comment:
Since this is following https://github.com/liquality/chainabstractionlayer formatting, perhaps it should also follow their type for derivation cache: `type DerivationCache = { [index: string]: Address }`

https://github.com/liquality/chainabstractionlayer/blob/dev/packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.ts#L24